akka-http-core/src/main/scala/akka/http/impl/util/package.scala

```diff
@@ -127,6 +127,53 @@ package object util {
 
 package util {
 
+  import akka.http.scaladsl.model.{ ContentType, HttpEntity }
+  import akka.stream.{ Outlet, Inlet, FlowShape }
+  import scala.concurrent.duration.FiniteDuration
+
+  private[http] class ToStrict(timeout: FiniteDuration, contentType: ContentType)
```

Comment:
I've seen this just now. I wonder if we aren't missing a kind of half-the-way abstraction that more closely follows the old `TimerTransform` if we need to double the lines of code in this change for such a simple thing?